src/atoms/Heading/Heading.js

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
 
 const Heading = (props) => {
-  const classes = classNames({
-    hdg: true,
-    [`hdg-${props.level}`]: props.level,
-    [props.elementClass]: props.elementClass
-  });
+  // This component is for semantically providing h1 - h6
+  // depending on the html document outline.
+  // By default the `hdg-*` class will match the level,
+  // but to visually change the size, pass in classes
+  // through the `elementClass` prop
+
+  const classes = () => {
+    if (props.elementClass) {
+      return props.elementClass;
+    } else {
+      return `hdg hdg-${props.level}`;
```

Comment:
I removed `classNames` because I think the string template is easier to understand in this case